### ISSUE TITLE
Improve developer resources for Google Group

### ISSUE THREAD

**Issue by samreid** [phetsims/phet-info#163]
@pixelzoom reported that there are some questions that are asked repeatedly in the Google Group.  Particularly about how to get started.  How can PhET better support our clients for this?

* FAQ?
* Improve example sim? See https://github.com/phetsims/example-sim/issues/13
* Monthly webinar?  Clients can share their screens, ask questions,...
* ???

PhET will need to evaluate how many people this will serve (Google Group traffic is relatively low), and what resources would be involved.  This will involve @kathy-phet.

@pixelzoom volunteered to be assigned to this issue, but acknowledged that he wouldn't be able to get to it for a while.

**Comment by samreid** [comment]
@pixelzoom also volunteered to create an issue in example-sim for those recommendations?

**Comment by pixelzoom** [comment]
example-sim was significantly improved in phetsims/example-sim#13 and has already benefited one Google Group member. 

Re the other suggestions...

FAQ: My post https://groups.google.com/g/developing-interactive-simulations-in-html5/c/jNXvJwkXj7k seems to have become a defacto "how to get started" FAQ.  It feels like I refer someone to it once a week.  Putting together a list of FAQ questions and answers would probably take 3-6 hours, and would require @kathy-phet's approval, prioritization, and assignment of a developer.  (And I'm not too excited to take this on right now.) 

Webinar: I'm skeptical that there are enough interested Group members to make a webinar worthwhile, but I might be wrong. A first step would be to ask @kathy-phet if there are resources for a webinear.  A first second would be to have someone from PhET post to the Group to see how many people would be interested.  (I've not interested in being the webinar instructor.)

A common theme in the above is that this is going to take resources, and requires input/approval from @kathy-phet.  So I'm going to label this for developer meeting, and add to the "requires Kathy" part of the dev meeting agenda.

**Comment by samreid** [comment]
From today's meeting: we had some successes from this group, from the x-ray-diffraction development, statistics sim, etc.  However, we don't want to spend a disproportionate amount of time boosting these resources.  It's been very rare for a community member to contribute back improvements to our common code libraries.  Our code libraries require a minimum amount of skill and proficiency before it makes sense.

@kathy-phet asks for voices on how our project is going as open source.

@pixelzoom says: The google group hasn't been very useful overall.  Is it worth having the group if we aren't putting a significant amount of effort into it?  Most of the questions are "How do I get started?"

@jonathanolson says: If we want to help 3rd parties, we should spend more on resources for 3rd parties. We don't have so many tutorials for getting people started.  We don't have easily browsable documentation.  Is there something we can do that will help for onboarding new team members, and also helps open source developers?

@kathy-phet says our investment of onboarding close collaborators has been very valuable and insightful, e.g. with @chrisklus recent training.  Moreso than the google group.  Maybe we would focus on interns and ramping up rather than the google group, and as we make things easier to onboard, it will be easier for 3rd parties in general.

@pixelzoom identifies that "master is unstable" pattern is not amenable to a broad community usage.

@kathy-phet says the Google Group and that level of support is not currently funded.

@zepumph says: We do get error reports from the community, when we make a breaking change that doesn't work for all environments.

@jonathanolson says: We develop libraries that aren't locked in to PhET, they are more general.

@kathy-phet asks if Oliver or the phet-help manager can be an intermediary.

@samreid asks for clarification: Are we using up too much time on the Google Group as it is now?

@pixelzoom says it made sense to upgrade example-sim for @chrisklus and his learning group, so it is more representative of a typical sim.

@samreid says he feels we are missing out on a tremendous opportunity as an open source project, and does not have a good understanding as to why it hasn't yet "taken off", and what we stand to gain if one day it gains more traction.

@zepumph asks if we can have Oliver or a non-developer be the first responder for the google group, so we aren't using developer hours on that unnecessarily.  Somewhat like he triages questions to phethelp?

@pixelzoom: it sounds reasonable to have this first responder to make sure questions are handled appropriately.  I don't mind doing it, but perhaps it could be done more efficiently.  Most questions are "how do I get started" which could have a boilerplate response.  The other questions are "help me design a [...] sim".

**Comment by samreid** [comment]
We discussed this again briefly today, @oliver-phet mentioned he is happy to step in and help here, but said it would be more efficient to restructure our resources--adding a FAQ, streamlining that page.  Maybe even moving away from Google Groups if it is making it harder to reach this sustainable goal.

@oliver-phet also mentioned GitHub has a feature called "Discussions", which may be a good alternative if we move away from Google Groups.  We would need to enable it on a repo-by-repo basis.  @oliver-phet said discussions for sim repos could replace unfuddle, or the phethelp email, etc.  May be able to do Google Group-like discussions.

**Comment by jessegreenberg** [comment]
Discussed 8/19/21 - 

@oliver-phet mentioned that historically we created this group to make it so that it wasn't necessary to go through phetthelp or an administrator to ask questions of developers. Assigning an administrator to the google group feels like the wrong direction in that regard.

@oliver-phet said it was very easy to update the "welcome message" for the google group, it would be easy to add FAQ or other resources right there to see if it cuts down on the amount of repeated questions we have to answer.

Ran out of time again, will discuss next time.
